lib/beacon_web/controllers/api/page_controller.ex

```diff
@@ -1,10 +1,15 @@
 defmodule BeaconWeb.API.PageController do
   use BeaconWeb, :controller
-
   alias Beacon.Content
+  alias Beacon.Repo
 
   action_fallback BeaconWeb.API.FallbackController
 
+  def index(conn, _params) do
+    pages = Content.list_pages(:dev) |> Repo.preload(:components)
```

Comment:
```suggestion
    pages = 
      :dev
      |> Content.list_pages()
      |> Repo.preload(:components)
```
When possible, pipelines should always start with a literal, as opposed to a function call